### DISCUSSION TITLE
Is there a way to customize the name shown in the dashboard?

### DISCUSSION THREAD

**Discussion by paulomorgado** [microsoft/aspire#6755] [Q&A]
I'd like to have a different name other than app host assembly name minus `.AppHost`.

**Comment by davidfowl** [comment]
Nope.

**Comment by IEvangelist** [comment]
What name are you referring to? You can configure the `Dashboard:ApplicationName`, see: https://learn.microsoft.com/en-us/dotnet/aspire/fundamentals/dashboard/configuration?tabs=bash#other

**Reply by davidfowl** [reply]
I stand corrected.

**Reply by paulomorgado** [reply]
@IEvangelist, that seems to apply to the standalone dashboard.

Setting that configuration on the app host has no effect on the text that appears in the dashboard.

Neither does setting `builder.Environment.ApplicationName = "My System"`.

The only thing that seems to work is setting the assembly name on the app host project to `<AssemblyName>My System</AssemblyName>`.
